packages/main/src/themes/SideNavigationItem.css

```diff
@@ -0,0 +1,87 @@
+:host(:not([hidden])) {
+    display: inline-block;
+    width: 100%;
+    height: 100%;
+    background: var( --sapList_Background);
```

Comment:
Please remove the space as we often search the code with regex for var(--SOMETHING) and we may miss this ;)